org.opent2t.sample.thermostat.superpopular/com.wink.thermostat/js/package.json

```diff
@@ -19,12 +19,12 @@
   },
   "dependencies": {
     "opent2t": "1.0.1",
-    "opent2t-translator-helper-wink": "*"
+    "opent2t-translator-com-wink-hub": "*"
```

Comment:
There should be an equivalent change in the nest thermostat package.json, right?